local-build.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-BRANCH=test
+BRANCH="test"
 
 rm -f tech.feliciano.pocket-casts.flatpak
-rm -rf _build ; mkdir _build
-rm -rf _repo ; mkdir _repo
+rm -rf build-dir ; mkdir build-dir
```

Comment:
I don't think those changes are necessary.

Reply:
I tend to use `build-dir` myself but I can revert these changes.